pwa-devdocs/src/tutorials/hello-upward/using-template-resolver/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Using the TemplateResolver
+---
+
+In this tutorial, you will learn how to use the [TemplateResolver][] in your UPWARD specification file to serve an HTML page rendered server-side.
+
+This tutorial builds on the project described in the previous [Creating a simple server][] tutorial.
+
+## Create web page templates
+
+In your project directory, create a `templates` directory with [Mustache][] template files for the different pieces of the HTML page.
+
+**`open-document.mst`**
+
+```mustache
+<!DOCTYPE html>
+<html language="en">
+
+  <head>
+    <meta charset="utf-8">
+```
+
+This template partial creates the beginning of the HTML response.
+The `head` tag is left open to allow custom metadata, such as title, scripts, and styles.
+
+**`open-body.mst`**
+
+```mustache
+  </head>
+  <body>
+```
+
+This template partial closes the `head` tag and begins the `body` tag of the HTML response.
+
+**`close-document.mst`**
+
+```mustache
+  </body>
+</html>
+```
+
+This template partial closes the `body` and `html` tag for the HTML response.
```

Comment:
I think the partials are overkill for this tutorial.

Since this isn't a tutorial on Mustache, I think the `hello-world.mst` template could just contain all of this.

Reply:
I agree that partials add some complexity for a simple tutorial. However, this pattern is used in the Venia implementation. Readers will undoubtedly draw comparisons between what's in the Venia source and how we're telling them to do something.